tests/ship_test.py

```diff
@@ -132,7 +132,7 @@
             try:
                 statuses = json.loads(" ".join(lines))
             except json.decoder.JSONDecodeError as er:
-                Utils.errorExit("javascript client output was malformed in %s. Exception: %s" % (shipClientErrorFile, er))
+                Utils.errorExit("ship_client output was malformed in %s. Exception: %s" % (shipClientErrorFile, er))
```

Comment:
Would prefer f-strings when updating. I'll only comment here on this one so if you choose not to update, there is only one comment to close.

Reply:
Since only minor changes in this python script, did not update to using f-string.